lib/va_profile/models/telephone.rb

```diff
@@ -2,11 +2,14 @@
 
 require_relative 'base'
 require 'common/models/attribute_types/iso8601_time'
+require 'va_profile/concerns/defaultable'
```

Comment:
Adding this for consistency. It usually ends up not mattering because one of the other models has probably already required it, but this could cause errors under some circumstances.